haematologica.csl

```diff
@@ -0,0 +1,210 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="en-US">
+  <info>
+    <title>Haematologica</title>
+    <id>http://www.zotero.org/styles/haematologica</id>
+    <link href="http://www.zotero.org/styles/haematologica" rel="self"/>
+    <link href="http://www.zotero.org/styles/medicinal-research-reviews" rel="template"/>
+    <link href="http://www.haematologica.org/information-for-authors" rel="documentation"/>
+    <author>
+      <name>Patrick O'Brien</name>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <issn>0390-6078</issn>
+    <eissn>1592-8721</eissn>
+    <updated>2017-07-20T14:59:46+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale>
+    <terms>
+      <term name="page" form="short">
+        <single>p</single>
+        <multiple>p</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author" suffix=". ">
+      <name sort-separator=" " initialize-with="" name-as-sort-order="all" delimiter=", " delimiter-precedes-last="always"/>
+      <label form="short" prefix=" (" suffix=")" strip-periods="true"/>
```

Comment:
unless there's a specific example, I'd expect the editor format to be the same as for book chapters, i.e. , editor(s)